backup.py

```
@@ -217,33 +331,38 @@ def restore_container_appdata(backup_root, container_id, dest_path, host, ssh_us
     try:
         container = docker_client.containers.get(container_id)
         config_data = container.attrs
         with json_path.open('w') as f:
             json.dump(config_data, f, indent=2)
         logger.info(f"Saved config for {container_id} to {json_path}")
+        return True
     except docker.errors.NotFound:
         logger.warning(f"Container {container_id} not found.")
+        return False
     except docker.errors.APIError as e:
         sub = f"Backup error"
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)